api/healthcheck.go

```diff
@@ -31,6 +31,7 @@ type HealthChecker struct {
 	CertificateManager insolar.CertificateManager
 	NodeNetwork        network.NodeNetwork
 	PulseAccessor      pulse.Accessor
+	// todo: use insolar.HealthChecker
```

Comment:
violated 7.5 in https://insolar.atlassian.net/wiki/spaces/DEV/pages/738131969/Go+code+style#Gocodestyle-4Interfaces